plugins/inputs/influxdb_v2_listener/influxdb_v2_listener.go

```diff
@@ -0,0 +1,396 @@
+package influxdb_v2_listener
+
+import (
+	"compress/gzip"
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"net"
+	"net/http"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	tlsint "github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"github.com/influxdata/telegraf/plugins/parsers/influx"
+	"github.com/influxdata/telegraf/selfstat"
+)
+
+const (
+	// defaultMaxBodySize is the default maximum request body size, in bytes.
+	// if the request body is over this size, we will return an HTTP 413 error.
+	defaultMaxBodySize = 32 * 1024 * 1024
+)
+
+type InfluxDBV2Listener struct {
+	ServiceAddress string `toml:"service_address"`
+	port           int
+	tlsint.ServerConfig
+
+	ReadTimeout  internal.Duration `toml:"read_timeout"`
+	WriteTimeout internal.Duration `toml:"write_timeout"`
+	MaxBodySize  internal.Size     `toml:"max_body_size"`
+	MaxLineSize  internal.Size     `toml:"max_line_size"` // deprecated in 1.14; ignored
```

Comment:
I'd remove this. if it's already deprecated, there's no reason to add it to a new plugin.

Reply:
Perfect, missed this, will remove.